tests/e2e_tests/run

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+set -e
+
+COMPOSE_CMD="docker-compose -p gui-tests \
+             -f ${INTEGRATION_PATH}/docker-compose.yml \
+             -f ${INTEGRATION_PATH}/docker-compose.demo.yml \
+             -f tests/e2e_tests/docker-compose.e2e-tests.yml"
+
+# by default just add minio, with COMPOSE_CMD_BASE this creates the standard onprem ST setup
+COMPOSE_FILES_DEFAULT=( "${INTEGRATION_PATH}/docker-compose.storage.minio.yml" )
+COMPOSE_FILES=()
+
+usage() {
+    echo "runner script for gui-specific integration tests"
+    echo ""
+    echo "./gui-tests"
+    echo -e "\t-h --help"
+    echo -e "\t-c --skip-cleanup \tleave containers running after tests"
+    echo -e "\t-f=<FILE>         \tspecify custom compose file(s); default files will not be used,"
+    echo -e "\t                  \tmake sure to specify all files you need"
+    echo -e "\t other args will be passed to the testing container's py.test command"
```

Comment:
The container is not using `py.test` (which I don't know why).

Two options, we make the container to actually use `pytest`, or at least we modify here the comment.